modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -156,8 +156,17 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::LAROMANCEStressUpdateBaseTempl(
     _extrapolation(this->template declareProperty<Real>("ROM_extrapolation")),
 
     _derivative(0.0),
-    _old_input_values(3)
+    _old_input_values(3),
+    _wall_dislocations_step(this->template declareGenericProperty<Real, is_ad>(
+        this->_base_name + "wall_dislocations_step")),
+    _cell_dislocations_step(this->template declareGenericProperty<Real, is_ad>(
+        this->_base_name + "cell_dislocations_step")),
+    _plastic_strain_increment(RankTwoTensor::Identity()),
```

Comment:
why is it initialized as Identify instead of zero?

Reply:
Because I was confused about the RankTwoTensor initialization options. I assume `::initNone` is the right option here.